### DISCUSSION TITLE
잠시 후가 또 떠요

### DISCUSSION THREAD

**Discussion by eonnnn** [changdoc/naver-vaccine-macro#61] [Q&A]
1.0.9 버전 2000ms 간격으로 창 하나만 켜서 시도했는데
잠시 후 다시 시도하세요가 떴어요
다른 기계나 브라우저로 접속하지 않았어요
어제랑 오늘 아침까지는 괜찮았는데 갑자기 뜨네요..

**Comment by chaeun9811** [comment]
저는 오늘오전11시부터 지금까지는 문제없이 잘작동하고있네요

**Comment by OneTop4458** [comment]
https://github.com/changdoc/naver-vaccine-macro/discussions/51

참고 바라며 지속 발생 시 인터벌은 2500ms로 조절 후 시도 바랍니다.
